.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
please remove IDE specific files from repo and add `.vscode/` pattern to the `.gitignore` file

Reply:
I removed IDE files and I added .vscode to .gitignore